core/mondoo-linux-security.mql.yaml

```diff
@@ -1136,16 +1136,16 @@ queries:
         ```
     query: |
       if( file("/boot/grub2/grub.cfg" ).exists) {
-        file("/boot/grub2/grub.cfg").content.lines.where( _ == /^[^#]/ ).contains("audit\=(\s+)?1")
+        file("/boot/grub2/grub.cfg").content.contains('audit=1')
       }
       if( file("/boot/grub/grub.cfg").exists ) {
-        file("/boot/grub/grub.cfg").content.lines.where( _ == /^[^#]/ ).contains("audit\=(\s+)?1")
+        file("/boot/grub/grub.cfg").content.contains('audit=1')
```

Comment:
How about `file("/boot/grub/grub.cfg").content.lines.where( _ == /^[^#]/ ).any(_ == /audit(\s+)?\=(\s+)?1/)` @micheelengronne  ?

This would use a regex pattern, that will exclude lines which begin with a `#` and also take into account none or more white space characters between `audit` `=` and `1`, to take into account the possibility of different formatting.